app/config.py

```diff
@@ -0,0 +1,5 @@
+import os
+
+class Config:
```

Comment:
likewise, config stuff should be in main already - a local merge will fix this

Reply:
I've done a local merge and it resolved most of the merge conflicts, there's just a few easy-to-fix conflicts left now.